src/components/Verse/OverflowVerseActionsMenuBody/ShareVerseActionsMenu/index.tsx

```diff
@@ -1,26 +1,32 @@
-import React, { useState, useEffect } from 'react';
+/* eslint-disable max-lines */
+/* eslint-disable react-func/max-lines-per-function */
+import React, { useContext, useEffect, useState } from 'react';
 
 import clipboardCopy from 'clipboard-copy';
 import { useRouter } from 'next/router';
 import useTranslation from 'next-translate/useTranslation';
-import { useSelector, shallowEqual } from 'react-redux';
+import { useSelector } from 'react-redux';
+
+import copyVerse from '../../AdvancedCopy/utils/copyVerse';
```

Comment:
nit: we should do `@/components/Verse/AdvancedCopy/utils/copyVerse` instead of `../../`